### PR TITLE
Fix/improve dynamicIdle hidding

### PR DESCRIPTION
**Pull request by asizon**
There is an issue since we removed locale string for dynamic idle disabled, so i have hidde it for all versions since dynamicIdle exist in configurator(1.43).

### CODE REVIEW

File: src/js/tabs/motors.js
Comment (haslinghuis) [inline src/js/tabs/motors.js:773]: We could further simplify using 

```suggestion
            $('div.digitalIdlePercent').toggle(protocolConfigured && digitalProtocol && !FC.ADVANCED_TUNING.idleMinRpm && !FC.MOTOR_CONFIG.use_dshot_telemetry);
```

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

File: src/js/tabs/motors.js
Comment (asizon) [reply src/js/tabs/motors.js]: good!simplified it, use_dshot_telemetry should be avoid here

**Comment by asizon** [comment]
I have improved idle min parametre phrase when it is disabled, thought that It is more appropriate and avoids OFF word repeating.

**Comment by asizon** [comment]
@haslinghuis sorry but we cant simplied it since it need diferent way to hide it, with your proposal Idle box will not appears after dshot bidir disabling because doesnt work with toggle way.

**Review by haslinghuis — APPROVED** [review]
||  😉  👍

**Review by nerdCopter — APPROVED** [review]
* approving untested; assuming it gates correctly on older firmware if parameters do not exist

**Comment by asizon** [comment]
> || 😉 👍

wait please, i think finnaly we cant do it :(

**Comment by asizon** [comment]
@haslinghuis with || we cant go with other non digital esc protocols, digitalIdle is always here

**Comment by haslinghuis** [comment]
It would be nice to display (maybe even able to edit also) the idleMinRpm (will show active PID profile value) in motor tab when use_dshot_telemetry is enabled and idleMinRpm != 0

**Comment by asizon** [comment]
> It would be nice to display (maybe even able to edit also) the idleMinRpm (will show active PID profile value) in motor tab when use_dshot_telemetry is enabled and idleMinRpm != 0

Interesting..maybe we can discuss for othrr PR?

**Pull request merged**
